src/dataDisplay/personCoreMilestones.jsx

```diff
@@ -113,7 +113,12 @@ const useStyles = makeStyles((theme) => {
         boxShadow: 'none',
         opacity: 1,
     };
-    const iconClassSize = 24;
+
+    /**
+     * That is the base size for the class icon (which isn't an SVG), so that the size can be
```

Comment:
Nitpick/NAB: Should we capitalize CLASS and/or do something else to make sure it's clear we're talking about CLASS 101 et al / the baseball diamond thingie, versus, say, the HTML `class` attribute and/or some other technical meaning of the word "class"?

Reply:
I changed this to `basesClassIconSize`

Reply:
I meant in the comments.  Probably a NAB.

Reply:
Fixing...

Reply:
Comment tweaked accordingly in 207cc05.  Resolving.